src/supertux/console.cpp

```diff
@@ -384,7 +385,7 @@ Console::autocomplete()
   // depending on number of hits, show matches or autocomplete
   if (cmds.empty())
   {
-    m_buffer.addLines("No known command starts with \"" + prefix + "\"");
+    m_buffer.addLines(_("No known command starts with \"") + prefix + _("\""));
```

Comment:
Ah, this is a hidden feature or similar. I get it ok. I'll drop this from the patch. How about the rest, do you feel that is makes sense?